arraycontext/__init__.py

```diff
@@ -31,12 +31,13 @@
 import sys
 from .context import ArrayContext, DeviceArray, DeviceScalar
 
-from .transform_metadata import (CommonSubexpressionTag,
-        ElementwiseMapKernelTag)
-
-# deprecated, remove in 2022.
+# deprecated, remove in 2022
 from .metadata import _FirstAxisIsElementsTag
 
+from .transform_metadata import (CommonSubexpressionTag,  # noqa: F401
+        ElementwiseMapKernelTag, ParameterValue,
+        IsDOFArray)
```

Comment:
Please organize your diffs so that the changes are clearly visible.

Reply:
Cleaned up this change.